auth0/auth0-deploy-cli
# Bugfix/issue 875

### 🔧 Changes

- Added `sink.azurePartnerTopic` as a field to be stripped when creating a log stream
- Fixes #875 

### 📚 References

- [Create a log stream - Auth0 Management API v2](https://auth0.com/docs/api/management/v2/log-streams/post-log-streams)

### 🔬 Testing

- Covered by unit test
- Manual testing: 
1. Import log stream of the format
```yaml
logStreams:
  - name: Azure Event Grid
    filters:
      - type: category
        name: auth.token_exchange.fail
      - ...
    sink:
      azureSubscriptionId: <subscription id>
      azureRegion: <region>
      azureResourceGroup: <resource group>
      azurePartnerTopic: <partner topic name>
    status: active
    type: eventgrid
```
2. Create operation should succeed

### 📝 Checklist
